css-two-column-layout/styles.css

```diff
@@ -0,0 +1,78 @@
+html,
+body {
+  font-family: Karla, sans-serif;
+}
+
+h1 {
+  margin-bottom: 18px;
+  margin-top: 60px;
+  letter-spacing: 5px;
+  font-weight: 700;
+  font-size: 38px;
+}
+
+h5 {
+  margin-top: 22px;
+  margin-bottom: 0;
+  color: rgb(39, 39, 39);
+  letter-spacing: 1px;
+  font-size: 12px;
+}
+
+p {
+  margin-top: 10px;
+  line-height: 25px;
+  color: gray;
+  font-size: 0.8rem;
+  font-weight: 400;
+}
+
+button {
+  letter-spacing: 1px;
+  font-weight: 700;
+  color: #ec5d57;
+  background-color: white;
+  border: 2px solid #ec5d57;
+  padding: 10px 40px;
+}
+
+.container {
+  margin: auto;
+  max-width: 1000px;
+}
+
+.row {
+  display: flex;
+  justify-content: center;
+  flex-wrap: wrap;
+  margin-bottom: 25px;
+}
+
+.column-full {
+  text-align: center;
+  width: 100%;
+}
+
+.column-half {
+  max-width: 50%;
+}
+
+.icon {
+  height: 100px;
+}
+
+.column-three-fourth {
+  padding-left: 5px;
+  padding-top: 10px;
+  flex-basis: 75%;
+}
+
+.column-one-fourth {
+  display: flex;
+  justify-content: flex-end;
+  flex-basis: 25%;
+}
```

Comment:
The property you are using for width varies between each of your column classes, please choose one property and make this consistent. Also, make sure that column classes only include a `width` or `flex-basis`, anything in addition to that should be applied to separate classes.